src/base/settingsstorage.cpp

```diff
@@ -26,7 +26,9 @@
  * but you are not obligated to do so. If you do not wish to do so, delete this
  * exception statement from your version.
  */
+#include "settingsstorage.h"
```

Comment:
> This "good technique" is almost entirely based on opinions, rather than facts, references, or specific expertise.

Thank you for your opinion on this topic, but I did ask another question. If you have time, please, explain me how the rules, supported by you, make possible to inspect a code faster. If there is a rational explanation (like in [the Google code guide](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes) for example), I will be thankful for that.